dissect/target/plugins/os/windows/network.py

```diff
@@ -224,11 +225,13 @@ def _try_value(subkey: RegistryKey, value: str) -> str | list | None:
         return None
 
 
-def _get_config_value(key: RegistryKey, name: str) -> set:
+def _get_config_value(key: RegistryKey, name: str, split_char: list[str] | None = None) -> set:
```

Comment:
```suggestion
def _get_config_value(key: RegistryKey, name: str, sep: str | None = None) -> set:
```
Rename to something that's also used in Python standard library, plus we can just make this a string since you're treating it like a list of single character strings anyway. This is also in line how Pythons own "split" is used.